cmd/envtool/envtool.go

```diff
@@ -347,6 +348,40 @@ func rmdir(paths ...string) error {
 	return errs
 }
 
+// cat will show the content of a file.
+func cat(paths ...string) error {
+	currentDir, err := os.Getwd()
```

Comment:
There is no need to do that – `os.Open` opens files in the current directory by default